### PR TITLE
Feature more options to search box

### PR DESCRIPTION
**Pull request by eyalvo-matrix**
ADD MORE STYLES TO THE MAIN SEARCH BOX

### CODE REVIEW

File: library/globals.inc.php [hunk 1]
```diff
@@ -288,6 +288,8 @@ function gblTimeZones()
             array(
                 'dual' => xl('Dual'),
                 'comprehensive' => xl('Comprehensive'),
+                'fixed' => xl('fixed'),
```
Comment (bradymiller) [inline library/globals.inc.php]: Capitalize Fixed and capitalize None

File: library/globals.inc.php [hunk 2]
```diff
@@ -288,6 +288,8 @@ function gblTimeZones()
...
+                'none' => xl('none'),
             ),
             'dual', // default
             xl('Search Patient By Any Demographics, Dual additionally lets direct access to Patient Finder, Comprehensive has collapsed input box')
```
Comment (bradymiller) [inline library/globals.inc.php]: Provide a quick explanation to the Fixed and None setting in line above.

File: library/globals.inc.php (hunk 1 above)
Comment (bradymiller) [reply library/globals.inc.php]: For cap, I only mean the labels that are being translated.

**Comment by bradymiller** [comment]
hi @eyalvo-matrix , This looks reasonable and just a couple minor things that need to be addressed. thanks! -brady

File: library/globals.inc.php (hunk 1 above)
Comment (eyalvo-matrix) [reply library/globals.inc.php]: Of course thanks

File: library/globals.inc.php (hunk 2 above)
Comment (eyalvo-matrix) [reply library/globals.inc.php]: sure

**Comment by bradymiller** [comment]
hi @eyalvo-matrix , Code looks good and bringing into the codebase. Thanks for the improvements! -brady

**Pull request merged**